scripts/ipfs/nodeinfo-ipfs

```diff
@@ -1,11 +1,18 @@
 #!/bin/sh
```

Comment:
`.sh` ext?

Reply:
It was removed because run-parts does not work with .sh extensions.... *shrug*

simply does not run them if it has a .sh ext.